.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - name: install openssl
         if: matrix.os == 'windows-latest'
         run: |
+          $ErrorActionPreference = "Stop"
```

Comment:
https://github.com/kube-rs/kube-rs/runs/5766982332?check_suite_focus=true failed to install OpenSSL, but this step didn't fail. Setting this _should_ stop it.
It's usually prepended automatically, but we have `fail-fast: false`.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference